funcx_endpoint/funcx_endpoint/endpoint/rabbit_mq/result_queue_publisher.py

```diff
@@ -17,8 +19,9 @@ class ResultQueuePublisher:
 
     def __init__(
         self,
+        *,
         endpoint_id: str,
-        pika_conn_params: pika.connection.Parameters,
+        conn_params: pika.connection.Parameters,
```

Comment:
Should we update the docstring as well?

Reply:
Good catch! :+1: I did this during the rebase.